src/Mvc/test/Mvc.FunctionalTests/HtmlGenerationWithCultureTest.cs

```diff
@@ -75,6 +75,7 @@ void ReadValuesFromDocument()
         }
     }
 
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/55930")]
```

Comment:
```suggestion
[QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/4907")]
```

Since we duped it. It might be worth preemptively quarantining `CacheTagHelper_AllowsVaryingByCulture` too.

Reply:
Done.